src/js/components/tasks/cards/card.tsx

```diff
@@ -137,7 +168,9 @@ const OrgCard = ({
             ownedByCurrentUser={ownedByCurrentUser}
             isCreating={isCreating}
             isDeleting={isDeleting}
-            isRefreshing={isRefreshing}
+            isRefreshingChanges={isRefreshingChanges}
```

Comment:
This should probably also get `isRefreshingOrg={isRefreshingOrg}` passed in, so it can display `i18n.t('Refreshing Org…')` similar to what it's currently doing for deleting/creating.